### PR TITLE
Feature/safari browser with cli

### PR DESCRIPTION
**Pull request by tgorskievo**

### CODE REVIEW

File: src/form-handlers/handler/checkbox.handler.js [hunk 1]
```diff
@@ -15,20 +15,18 @@ class CheckboxHandler {
   }
 
   handleFill(page, elementName, desiredValue) {
-    return page[elementName].filter(function (elem) {
-      return elem.element(by.xpath('..')).getText().then(function (text) {
-        return text === desiredValue;
+    return page[elementName].filter(elem => {
+      return elem.element(by.xpath('..')).getText().then((text) => {
+        return text.trim() === desiredValue;
```
Comment (tgorskievo) [inline src/form-handlers/handler/checkbox.handler.js]: For example, there is an element with value:
```
      Checkbox Option 3
      ```
`Google Chrome` and `Firefox` length is equal to 17 but in Safari it returns `43`. So the trim operation is done by default in those browsers.

Comment (sethii) [reply src/form-handlers/handler/checkbox.handler.js]: Nice catch!

File: docs-src/multibrowsering.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+## Multibrowsering
```
Comment (sethii) [inline docs-src/multibrowsering.md]: Cross-browser support :)

File: src/helpers/safari-browser-configurator.helper.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import shell from 'shelljs';
+import path from "path";
+
+export const safariBrowserConfigurator = (config) => {
+  shell.exec(`defaults write -app Safari DownloadsPath ${path.join(config.projectPath, config.downloads)}`);
```
Comment (sethii) [inline src/helpers/safari-browser-configurator.helper.js]: This has to be done through command line? There is no option to pass it with options?

Comment (tgorskievo) [reply src/helpers/safari-browser-configurator.helper.js]: Unfortunately, I did not see the possibility to configure it in the same way as for Firefox or Google Chrome

**Pull request merged**